src/bm_sim/tables.cpp

```diff
@@ -35,7 +36,7 @@ MatchActionTable::MatchActionTable(
 const ControlFlowNode *
 MatchActionTable::operator()(Packet *pkt) const {
   // TODO(antonin)
-  // this is temporary while we experiment with the debugger
+  //  this is temporary while we experiment with the debugger
```

Comment:
this also doesn't look tight

maybe the script wants: `// TODO(antonin): this is temporary while we experiment with the debugger`

Reply:
`tables.cpp:36:  Too many spaces before TODO  [whitespace/todo] [2]`
This is the particular complaint it has. Will put it all in one line.